src/Sylius/Bundle/ShopBundle/Resources/views/Cart/Summary/_item.html.twig

```diff
@@ -19,6 +19,6 @@
         <a href="{{ path('sylius_shop_cart_item_remove', {'id': item.id}) }}" class="ui circular icon button sylius-cart-remove-button"><i class="remove icon"></i></a>
     </td>
     <td class="right aligned">
-        <span class="sylius-total">{{ item.total|sylius_price(cart.currencyCode) }}</span>
```

Comment:
Actually these were correct, the issue here is that `currencyCode` of a cart is not updated on currency switch and that is what should be fixed by this PR 😉